build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.2'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
I don't know if this line breaks CI as well, if it does - it can be removed for now aswell